paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -145,15 +142,15 @@ inference_analysis_api_test_with_refer_result(test_analyzer_mobilenet_transpose
 
 # googlenet
 inference_analysis_api_test_with_fake_data(test_analyzer_googlenet
-  "${INFERENCE_DEMO_INSTALL_DIR}/googlenet" analyzer_resnet50_tester.cc "googlenet.tar.gz" false)
+  "${INFERENCE_DEMO_INSTALL_DIR}/googlenet" analyzer_resnet50_tester.cc "googlenet.tar.gz")
 
 # resnet50
 inference_analysis_api_test_with_fake_data(test_analyzer_resnet50
-  "${INFERENCE_DEMO_INSTALL_DIR}/resnet50" analyzer_resnet50_tester.cc "resnet50_model.tar.gz" true)
```

Comment:
Maybe set `true`-> `false` is enough?

Reply:
This flag is a huge workaround anyway, personally I don't believe we should use it at all, since MKL team has mentioned that its an undocumented feature and may disappear anytime. That's why I have removed it.

Reply:
Got it.